lib/App/Seacan.pm

```diff
@@ -100,7 +101,7 @@ sub install_cpan {
     my $cpanm_command = path( $self->seacan_perlbrew_root, "bin", "cpanm")->stringify;
     my $perl_command = $self->seacan_perl;
 
-    $, = " ";
+    local $OUTPUT_FIELD_SEPARATOR = q{ };
```

Comment:
It appears to me that I put `$,` there in the beginning (commit: 138d534b59ed21bfdd1c5ecd366281b8b1fd8acc ) -- but I don't see how that is relevant, since `system()` (and the underlying `exec()`) do not join its arguments.